onnxruntime/python/tools/transformers/gpt2_helper.py

```diff
@@ -501,13 +578,15 @@ def onnxruntime_inference(ort_session, inputs: Gpt2Inputs, total_runs: int = 0):
     def prepare_io_binding(ort_session, input_ids, position_ids, attention_mask, past, output_buffers, output_shapes):
         """ Returnas IO binding object for a session.
         """
-        return IOBindingHelper.prepare_io_binding(ort_session, input_ids, position_ids, attention_mask, past, output_buffers, output_shapes)
+        return IOBindingHelper.prepare_io_binding(ort_session, input_ids, position_ids, attention_mask, past,
+                                                  output_buffers, output_shapes)
 
     @staticmethod
     def get_outputs_from_io_binding_buffer(ort_session, output_buffers, output_shapes, return_numpy=True):
         """ Copy results to cpu. Returns a list of numpy array.
         """
-        return IOBindingHelper.get_outputs_from_io_binding_buffer(ort_session, output_buffers, output_shapes, return_numpy)
+        return IOBindingHelper.get_outputs_from_io_binding_buffer(ort_session, output_buffers, output_shapes,
```

Comment:
I think its a good idea to handle refactoring as diff PR.
It basically makes the change look bigger than it actually is.

Reply:
This line is changed by format tool (yapf) automatically.